bin/node/cli/src/cli.rs

```diff
@@ -62,13 +62,9 @@ pub struct RunCmd {
 	#[structopt(long = "disable-authority-discovery")]
 	pub authority_discovery_disabled: bool,
 
-	/// Maximum number of logs in a query.
-	#[structopt(long, default_value = "10000")]
-	pub max_past_logs: u32,
-
-	/// The dynamic-fee pallet target gas price set by block author
-	#[structopt(long, default_value = "1000000000")]
-	pub target_gas_price: u64,
+	#[allow(missing_docs)]
```

Comment:
Remove this allow and add some documentation instead.

Reply:
This is a flattened field. The user can not see this from the cli.